pkg/maps/ctmap/ipv6.go

```diff
@@ -85,8 +85,8 @@ func (k CtKey6) Dump(buffer *bytes.Buffer) bool {
 
 //CtKey6Global represents the key for IPv6 entries in the global BPF conntrack map.
 type CtKey6Global struct {
-	daddr   types.IPv6
 	saddr   types.IPv6
+	daddr   types.IPv6
 	sport   uint16
 	dport   uint16
 	nexthdr u8proto.U8proto
```

Comment:
`type CtKey6 struct` is the one that is incorrect, not `type CtKey6Global struct`.

The IPv4 case looks like the following:
```

type CtKey4 struct {
	daddr   types.IPv4
	saddr   types.IPv4
	sport   uint16
	dport   uint16
	nexthdr u8proto.U8proto
	flags   uint8
}

type CtKey4Global struct {
	daddr   types.IPv4
	saddr   types.IPv4
	sport   uint16
	dport   uint16
	nexthdr u8proto.U8proto
	flags   uint8
}

```

On bpf side, it looks:
```
struct ipv4_ct_tuple {
	__be32		daddr;
	__be32		saddr;
	/* The order of dport+sport must not be changed */
	__u16		dport;
	__u16		sport;
	__u8		nexthdr;
	__u8		flags;
} __attribute__((packed));
```

The Go IPv6 side looks like:
```
type CtKey6 struct {
	saddr   types.IPv6
	daddr   types.IPv6
	sport   uint16
	dport   uint16
	nexthdr u8proto.U8proto
	flags   uint8
}
type CtKey6Global struct {
	daddr   types.IPv6
	saddr   types.IPv6
	sport   uint16
	dport   uint16
	nexthdr u8proto.U8proto
	flags   uint8
}
```

And on bpf:
```
struct ipv6_ct_tuple {
	union v6addr	daddr;
	union v6addr	saddr;
	/* The order of dport+sport must not be changed */
	__u16		dport;
	__u16		sport;
	__u8		nexthdr;
	__u8		flags;
};
```
Therefore `CtKey6Global` must be correct and `CtKey6` not (latter is also the one that got changed in 088ff00e).

Good catch!